src/main/java/uk/ac/cam/cl/dtg/segue/api/EmailFacade.java

```diff
@@ -422,10 +382,17 @@ public Response generateEmailVerificationToken(@PathParam("email") final String
     @GZIP
     public final Response sendEmails(@Context final HttpServletRequest request,
 		    		@PathParam("contentid") final String contentId, 
-		    		@PathParam("emailtype") final Integer emailTypeInt, 
+		    		@PathParam("emailtype") final String emailTypeString,
```

Comment:
Can we just stick the enum type in instead of a String. Not sure if RestEasy will map it for us (or what happens if they provide something that isn't a valid enum type - presumably a standard error?) Happy either way, just a point of interest

Reply:
You get really awful unhandled (and unhandleable) errors if it doesn't map exactly, which is why I ended up with it this way round. I did try it that way at first!